src/mono/wasm/wasm.proj

```diff
@@ -34,6 +34,7 @@
     <ItemGroup>
       <WasmPInvokeModule Include="libSystem.Native" />
       <WasmPInvokeModule Include="libSystem.IO.Compression.Native" />
+      <WasmPInvokeModule Include="QCall" />
```

Comment:
Why is this needed?

Reply:
Because QCalls are implemented on the managed side as pinvokes with to the module 'QCall'.

Reply:
Wouldn't be better to handle that inside runtime?

Reply:
> Wouldn't be better to handle that inside runtime?

We want to make it possible to remove the native code of pinvokes/qcalls that are not called anywhere in managed.   If the runtime has an array of qcall functions, the native linker won't be able to drop